conf/messages

```diff
@@ -1871,6 +1871,6 @@ partner.telephoneNumber.hint                                        = Enter a UK
 partnerEmail.namedHeading                      = What is {0}’s email address?
 partnerEmail.para                              = Full details of how we use your information are in the
 partnerEmail.link                              = HMRC Privacy Notice
-partnerEmail.error.incorrect_format            = Enter an email address in the correct format, like name@example.com
+partnerEmail.error.incorrect_format            = Enter the email address in the correct format, like name@example.com
 partnerEmail.error.nothing_entered             = Enter the email address
-partnerEmail.error.incorrect_length            = The email address must be 132 characters or fewer
\ No newline at end of file
+partnerEmail.error.incorrect_length            = The email address must be 254 characters or fewer
```

Comment:
This is incorrect, according to the swagger doc (and the form validation) it is 132 characters